exasol/itde_manager.py

```diff
@@ -77,6 +82,14 @@ def bring_itde_up(conf: Secrets) -> None:
     conf.save(AILabConfig.cert_vld, "False")
 
 
+def _add_current_container_to_db_network(container_info: ContainerInfo):
+    network_name = container_info.network_info.network_name
+    docker_client = docker.from_env()
```

Comment:
don't forget to call `docker_client.close()` at the end. I remember that we had an issue in ITDE with too many unclosed docker clients....

Reply:
I saw in the test you are already using `ContextDockerClient`....

Reply:
good point